.gitignore

```diff
@@ -1,41 +1,7 @@
-demo
 dist
 node_modules
-npm-debug.log
-*.swp
-*.swo
-.DS_Store
```

Comment:
We need .DS_Store, it gets added by macOS when new files are added unfortunately